src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -46,9 +49,6 @@ export async function getCowSwapTradeQuote(
     ? fromAssetId(buyAsset.assetId).assetReference
     : COW_SWAP_NATIVE_ASSET_MARKER_ADDRESS
 
-  // making sure we do not have decimals for cowswap api (can happen at least from minQuoteSellAmount)
-  const normalizedSellAmountCryptoBaseUnit = normalizeIntegerAmount(sellAmount)
-
```

Comment:
this seems to be a requirement of logic surrounding minimums that has been removed. We never have decimals in the input amount

Reply:
this was rounding to 16 sig. figs. which was the root cause of the validation issues because the sell amount could be higher than the input amount -> send max means tx failure